template/web/wp-content/themes/<%= slug %>/src/css/_config.scss

```diff
@@ -0,0 +1,9 @@
+/*============================================================================*\
+  Config SCSS
+\*============================================================================*/
+
+/* Assets directories
+\*============================================================================*/
+
+$img-dir: '/static/img/';
+$svg-dir: '/static/svg/';
```

Comment:
`static` folder is in the theme directory right ? Does this work ? shouldn't it be `wp-content/theme/<%= slug %>/static/svg`